sympy/stats/joint_rv_types.py

```diff
@@ -188,11 +192,23 @@ def MultivariateT(syms, mu, sigma, v):
     Parameters
     ==========
 
-    syms: list/tuple/set of symbols for identifying each component
-    mu: A list/tuple/set consisting of k means,represents a k
-        dimensional location vector
+    syms: A symbol/str for identifying the random variable.
```

Comment:
```
syms: A symbol/str
    For identifying the random variable
```

Reply:
Similarly at other places.

Reply:
@czgdp1807 I will make the necessary changes